apps/web/src/pages/Pools/AssessmentPlanBuilderPage/AssessmentPlanBuilderPage.tsx

```diff
@@ -45,8 +45,8 @@ const pageTitle = defineMessage(messages.assessmentPlan);
 
 const pageSubtitle = defineMessage({
   defaultMessage:
-    "Select, organize and define the assessments used to evaluate each skill in the advertisement. Make sure every skill is assessed at least once to complete your assessment plan.",
-  id: "SSZY5w",
+    "Select, organize and define the assessments used to evaluate each skill in the advertisement. Make sure every essential skill is assessed at least once to complete your assessment plan.",
+  id: "iuA2pt",
   description: "Subtitle for the assessment plan builder",
```

Comment:
Orphaned French string for this is 

```
  "SSZY5w": {
    "defaultMessage": "Sélectionner, organiser et définir les évaluations ayant été utilisées pour évaluer chaque compétence de l’offre d’emploi. 
Veillez à ce que chaque compétence soit évaluée au moins une fois pour compléter votre plan d’évaluation.",
    "description": "Subtitle for the assessment plan builder"
  },
```
Do I resend the whole object for translation or does someone know what the French change should be?

Reply:
Here you go, use this:
`Sélectionner, organiser et définir les évaluations ayant été utilisées pour évaluer chaque compétence de l’offre d’emploi. Veillez à ce que chaque compétence essentielle soit évaluée au moins une fois pour compléter votre plan d’évaluation.`

Reply:
Thank you! 
https://github.com/GCTC-NTGC/gc-digital-talent/pull/10195/commits/cd9e0cedd0901745afc38eefd9f5cdac73e7559b